roles/aodh/tasks/monitoring.yml

```diff
@@ -1,20 +1,5 @@
 ---
```

Comment:
Since we are doing a significant refactor I think we should mimic as much as possible what the Site Controller team did.
Here is a good example:  https://github.blueboxgrid.com/blue-box-cloud/sitecontroller/blob/master/roles/percona/tasks/checks.yml
So `tasks/check.yml` will have something like:
```
---
- name: install mysql process check
  sensu_check_dict: name="check-mysql-process" check="{{ sensu_checks.percona.check_mysql_process }}"
  notify: restart sensu-client missing ok
  when: inventory_hostname in groups.percona_primary or inventory_hostname in groups.percona_secondary|default('[]')

- name: install garbd process check
  sensu_check_dict: name="check-garbd-process" check="{{ sensu_checks.percona.check_garbd_process }}"
  notify: restart sensu-client missing ok
  when: inventory_hostname in groups.percona_arbiter|default('[]')
```

Let's keep check installation in the role itself.

@Xiaohua-Shen do you have access to the sitecontroller repo ?